### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Setup_Linux_ScreenReader: On tab navigation to the # link beside heading "Cannot install .deb package due to "/etc/apt/sources.list.d/vscode.list: No such file or directory", it is read as "Cannot install .deb package due to"

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-docs#5545]
## Actual:
On tab navigation to the # link beside heading "Cannot install .deb package due to "/etc/apt/sources.list.d/vscode.list: No such file or directory", it is read as "Cannot install .deb package due to". The full name is not read.

**Similar issue is observed with Chrome + JAWS.**

### **Similar issue is observed in following scenario:**
1. Docs Setup_MacOS: NVDA is not announcing the link name "Why do I see" given beside heading "Why do I see "Visual Studio Code would like access to your calendar." descriptively.

## Expected:
NVDA should announce the full name of the link. It should read the link name as "Cannot install .deb package due to "/etc/apt/sources.list.d/vscode.list: No such file or directory".

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win10;#FTP;#DesktopWeb;#A11ySev2;#WCAG1.3.1;#NVDA;#JAWS;#Chrome;#STP;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win10
Chrome Version: Version 103.0.5060.134(Official build)(64-bit)
ScreenReader: NVDA 2022.2 / JAWS 2022.2112.24 ILM

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab till "Docs" link.
4. From left navigation, select "Setup".
5. Tab till "Linux" link and press enter.
6. Tab till the link given beside heading "Cannot install .deb package due to "/etc/apt/sources.list.d/vscode.list: No such file or directory"
7. Turn on NVDA.
8. Observe that NVDA is announcing the link name as "Cannot install .deb package due to"

## User Impact:
Screen reader users will not be able to know the full name of the link.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment
![LINKNAME](https://user-images.githubusercontent.com/93374043/183046832-9d7e5db7-79a4-4ad0-bb84-f6b6927a3877.gif)

![JAWS LINKNAME](https://user-images.githubusercontent.com/93374043/183050790-c63945df-71d2-48b1-bdff-3f1f63373edf.gif)

**Comment by gregvanl** [comment]
Closing as external as this appears to be an issue with the JAWS screenreader. The Markdown and rendered text are correct.
Closing as no "Resolved" state for GitHub issues.

**Comment by Yashi1201** [comment]
@gregvanl Here, the issue is repro with NVDA also. When tab focus goes to the link, both screen reader reads it as "Cannot install .deb package due to". Screen reader is not reading full name of the link.
![5545_issue repro](https://user-images.githubusercontent.com/93374043/192735549-f06e4b92-f1cb-49de-82d2-76fc97893c8a.png)

Code snippet image : 
![5545_code snippet image](https://user-images.githubusercontent.com/93374043/192736767-a71dc6f4-a1bc-427d-a0e9-91ba734e17f3.png)

Could you please reopen the bug and fix this issue or let us know to which it is external to. Thanks

**Comment by Yashi1201** [comment]
Closing as issue is not repro in env https://code.visualstudio.com/
![5545_closing](https://user-images.githubusercontent.com/93374043/193750518-2e889019-8567-448c-a5aa-f843385b57a2.png)

Similar issue is not repro with JAWS also.
![5545_closing_JAWS](https://user-images.githubusercontent.com/93374043/193750576-bc654ef4-2a16-47d8-8395-39cbc75c117e.png)
